_rules/iframe-accessible-name-cae760.md

```diff
@@ -166,8 +167,17 @@ This `iframe` is not part of [sequential focus navigation][] because it has `tab
 <iframe tabindex="-1" role="presentation" src="/test-assets/SC4-1-2-frame-doc.html"> </iframe>
 ```
 
+#### Inapplicable Example 4
```

Comment:
I don't think this example should be here. Strictly speaking, focusable elements, which iframes **can** be, depending on the browser, can not have a semantic role of `presentation` or `none`. I only found this out recently. I think inapp example 3 covers this sufficiently.

Reply:
The ARIA in HTML document indicates presentation can be used. https://www.w3.org/TR/html-aria/.  What I'm trying to determine if if iFrames with tabindex-1 should be inapplicable from test regardless of role presentation -- or only when they have role presentation or none.  That is -- if the iFrame only has tabindex-1 but not a role is it a fail or inapplicable?  Seems like the iFrame still acts like a container or region of sorts -- so while it might not fit under user interface component under 4.1.2 it might fit under SC 1.3.1 information and relationships and still need an accessible name.